northstar/api/v1/lookup.py

```diff
@@ -0,0 +1,105 @@
+"""
+Lookup related routes
+"""
+# North Star ---  A lookup service for forged fed ecosystem
+# Copyright © 2021 Aravinth Manivannan <realaravinth@batsense.net>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+from urllib.parse import urlparse, urlunparse
+
+import requests
+from flask import Blueprint, jsonify, request
+
+from northstar.db import get_db
+from .errors import Error
+
+bp = Blueprint("API_V1_INTERFACE", __name__, url_prefix="/forge")
```

Comment:
CI build has failed. I think you have to choose a unique value for blueprint names/first parameter

Reply:
Noted. Changes will be made accordingly.